CPS-0010/README.md

```diff
@@ -122,16 +122,24 @@ Since its' authoring, CIP-30 has seen continued iteration with changes to its AP
 Since CIP-30's authoring there have only been two other competing standards, in CIP-45 | Decentralized WebRTC dApp-Wallet Communication ([#395](https://github.com/cardano-foundation/CIPs/pull/395)) and CIP-90 | Extendable dApp-Wallet Web Bridge ([#462](https://github.com/cardano-foundation/CIPs/pull/462/)).
 Although neither of of this standards have seen wide adoption.
 
-Although not a direct competitor, CIP-13 could be seen as an alternative standard, which fits some wallet connection niches.
-Where information is being transmitted to a wallet, to perform an action, using a novel URI scheme.
+With CIP-45 offering an alternative connection standard based on initiating a WebRTC connection via WebTorrent tracker.
+This is motivated by a want to diversify the connection standards in Cardano, taking a similar approach to that of [Wallet Connect](https://walletconnect.com/).
+Unlike CIP-30 this proposal does not define any standards for API shared after connection time, instead it just reuses the [CIP-30 Full API](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0030/README.md#full-api).
+
+CIP-90 is a daughter proposal of CIP-30, created around the time of the last major changes to CIP-30 within [#446](https://github.com/cardano-foundation/CIPs/pull/446).
```

Comment:
> a daughter proposal of CIP-30

Does CIP process provide a way to link proposals like that? If it's not formal, it's best to rephrase so that users are not confused about hierarchy of CIPs

Reply:
No, good spot.
Poor choice of wording, I was trying to just indicate that CIP90's content is largely just CIP30